sdk/template/Azure.Template/tests/Samples/Sample1_HelloWorld.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+#region Snippet:Azure_Template_Namespaces
+using Azure.Template.Models;
+#endregion
+using NUnit.Framework;
+
+namespace Azure.Template.Tests.Samples
+{
+    public partial class Sample1_HelloWorld: SamplesBase<TemplateClientTestEnvironment>
+    {
+        [Test]
+        [SyncOnly]
+        public void GettingASecret()
+        {
+            var endpoint = TestEnvironment.KeyVaultUri;
+
+            #region Snippet:Azure_Template_GetSecret
+#if SNIPPET
+            string endpoint = "https://myvault.vault.azure.net";
```

Comment:
This was already defined and won't compile. Instead, replace the definition of `endpoint` (including line 21 above) with code like this:

```csharp
    string endpoint = "https://myvault.vault.azure.net";
#if !SNIPPET
    endpoint = TestEnvironment.KeyVaultUri;
#endif
    SecretClient client = new SecretClient(new Uri(endpoint), new DefaultAzureCredential());
```